apps/ledger-live-mobile/src/components/RootNavigator/types/AccountsNavigator.ts

```diff
@@ -34,4 +34,15 @@ export type AccountsNavigatorParamList = {
   [ScreenName.Asset]: {
     currency: CryptoOrTokenCurrency;
   };
+  [ScreenName.AccountsList]: {
+    sourceScreenName: ScreenName;
+    showHeader?: boolean;
+    canAddAccount?: boolean;
+    isSyncEnabled?: boolean;
+  };
+  [ScreenName.AssetsList]: {
+    sourceScreenName: ScreenName;
+    showHeader?: boolean;
+    isSyncEnabled?: boolean;
+  };
```

Comment:
same non optional props & quite same optionals props 

You could create a type for this 2 screens :)

Reply:
I removed AssetsLists from there